rotkehlchen/api/v1/schemas.py

```diff
@@ -620,6 +620,7 @@ class HistoryEventSchema(
         SerializableEnumField(enum_class=HistoryBaseEntryType),
         load_default=None,
     )
+    customized_events_only = fields.Boolean(load_default=False)
```

Comment:
adjust the docs to explain how it works in the `api.rst` file

Reply:
Adjusted.